# Subscriber parameters 
subscriber_topic: /scan
queue_size: 1000

# Publisher parameters
twist_topic: /cmd_vel
twist_queue: 100

# Speeds
linear_speed: 0.1
angular_speed: 0.5

# Visualization Marker
marker_topic: visualization_marker
marker_queue: 100

# Server parameter
service_name: /Start_or_Stop_Server

# Range Client parameter
client_name: Observer

# Imu Client parameter
imu_topic: /imu
imu_client_name: Imu_Observer

# Deceleration
deceleration: -0.10